publication-testing/build.gradle

```diff
@@ -12,7 +12,7 @@ dependencies {
     api group: 'com.github.bibsysdev', name: 'apigateway', version: project.ext.nvaCommonsVersion
 
     api group: 'com.amazonaws', name: 'aws-java-sdk-lambda', version: project.ext.awsSdkVersion
-    api(group: 'com.github.javafaker', name: 'javafaker', version: '1.0.2') {
+    api(group: 'com.github.javafaker', name: 'javafaker', version: project.ext.javaFakerVersion) {
```

Comment:
changes in other modules other than create-message are limited to replacement of literals with variables. Refinement of each module will happen in own PR